### DISCUSSION TITLE
client recursive upload doesn't find assets

### DISCUSSION THREAD

**Discussion by teto** [immich-app/immich#13697] [Q&A]
### The bug

recursive submission doesn't seem to work with client: if I append a shell asterisk, then it can find the assets (jpg files)
```

$ immich upload -r 'Photos from 2018'
Crawling for assets...
No files found, exiting

$ pwd
/home/teto/Downloads/takeout-20230731T222132Z-001/Takeout/Google Photos

$ immich upload -r 'Photos from 2018'/*
Crawling for assets...
Checking files | ████████████████████████████████████████ | 100% | ETA: 0s | 1044/1044 assets
Found 255 new files and 789 duplicates
Uploading assets REDACTED....

$ ls -l 'Photos from 2018' | head
.rw-rw-r--  6.5M teto 31 juil.  2023 IMG_20180102_151535.jpg
.rw-rw-r--  8.3M teto 31 juil.  2023 IMG_20180102_152756.jpg
.rw-rw-r--  7.6M teto 31 juil.  2023 IMG_20180102_152812.jpg
.rw-rw-r--  6.9M teto 31 juil.  2023 IMG_20180102_154203.jpg


```

### The OS that Immich Server is running on

nixos-unstable ~ october 2024

### Version of Immich Server

v1.118.1

### Version of Immich Mobile App

dunno 

### Platform with the issue

- [ ] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
none I am using systemd services
```

### Your .env content

```Shell
same
```

### Reproduction steps

see first part

**Comment by bo0tzz** [comment]
For a google takeout you're much better off using https://github.com/simulot/immich-go, it'll clean up the metadata for you.

**Reply by teto** [reply]
thanks for the recommendation but I already have the images locally and I like uploading them at my pace. Also I would like to know immich CLI can't find pictures in some folders but not in this one. I've found issues about spaces in the directory path but they should have been fixed for my version.

**Comment by Aymendje** [comment]
This seems to be a bug when the filename contains spaces
Rename the folder to 'PhotosFrom2018' and it should work
